src/core/Akka.TestKit/TestKitBase_Receive.cs

```diff
@@ -721,7 +721,7 @@ private void PopPeekedEnvelope(MessageEnvelope envelope)
         {
             return ReceiveNAsync(numberOfMessages, cancellationToken)
                 .ToListAsync(cancellationToken)
-                .ConfigureAwait(false).GetAwaiter().GetResult();
+                .Result;
```

Comment:
We still need to use GetAwaiter().GetResult() because this automatically unwraps all AggregateException, using .Result here breaks behavior consistency.